docs/examples.rst

```diff
@@ -6,3 +6,21 @@ Ensure your device works with this simple test.
 .. literalinclude:: ../examples/csv_simpletest.py
     :caption: examples/csv_simpletest.py
     :linenos:
+
+Disklogger
+------------
+
+Logging data to .CSV file on CircuitPython Disk
+
+.. literalinclude:: ../examples/csv_disklogger.py
+    :caption: examples/csv_disklogger.py
+    :linenos:
+
+Dictwriter test
```

Comment:
Would you mind fixing this to `DictWriter` to match the name?  Thanks!

Reply:
Ok thanks. see https://github.com/tekktrik/CircuitPython_CSV/pull/18/commits/69a254e732be329b57c9364be094c3552ee00c3b